packages/design-system/src/ui-helpers/getModelInstanceTaskToolkit.tsx

```diff
@@ -178,12 +166,7 @@ export const getModelInstanceTaskToolkit = (task: string) => {
       return {
         getIcon: (className?: string) => {
           return (
-            <Icons.TextGenerationChat
-              className={cn(
-                "my-auto h-5 w-5 stroke-semantic-fg-primary",
-                className
-              )}
-            />
+            <Icons.TextGenerationChat className={className ?? "h-4 w-4"} />
           );
         },
         label: "Text Generation Chat",
```

Comment:
@orangecoloured This way the user of this component may need to think he might completely overwrite something. That is the reason why we usually have a default value and then user can partially overwrite it. This way we can reduce the mental overhead of user